CORTEX_M0+_RP2040/UsingCMSIS/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ target_include_directories(using_cmsis PUBLIC
         ${CMAKE_CURRENT_LIST_DIR}
         )
 
-target_compile_options( on_core_one PUBLIC
```

Comment:
This looks like it was just wrong. I assume this was copied from the `OnEitherCore` demo.